docs/docs/documentation/getting-started/installing-zelthy/manual.mdx

```diff
@@ -196,11 +208,24 @@ docker run --name zango_redis -d -p 6379:6379 redis
 
 ### Starting the celery worker
 
+For linux or MacOs:
+
 Perform the below steps from the root directory of the project
 
 ```
 celery -A <project_name> worker -l INFO
 ```
+For Windows:
+
+Perform the below steps from the root directory of the project
+
+```
+pip install gevent
+
+pip install eventlet
+
+celery -A <module> worker -l info -P eventlet
+```
```

Comment:
@rajxzel modify this section to indicate that either gevent or eventlet or a solo pool can be used